From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ms-prkhad__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

deneha commented:
@ms-prkhad Thank you for these edits.  It appears you may be MSFT internal.  Would you please update your profile to meet the [guidelines](https://github.com/Azure/azure-content/blob/master/contributor-guide/tools-and-setup.md#create-a-github-account-and-set-up-your-profile) so pull request reviewers can recognize you?

deneha commented:
@tamram Would you please review these edits and let me know if I have your approval to merge this request?  Thank you.

deneha commented:
At MSFT as prkhad - CLA not required

Commit:
added topic and images and added single TOC entry

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-revoke-enrollment-portal
Sync with master

Commit:
cleaned up a bit

Commit:
incorporated some feedback

Commit:
added preliminary group disable/delete shots and cleaned up some of the steps based on what is expected at GA

Commit:
interim add

Commit:
more changes

Commit:
added boxes to disable pngs, fixed TOC, modified topic

carolinacmoravia committed:
M15131: Extra ` that causing format issues

Hello, @iainfoulds,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version. There is an extra ` that should be deleted. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

Commit:
added explanatory material around X.509, cleaned up some language

Commit:
slight language fix

Commit:
moved X.509 explanatory content to security concepts article

Commit:
fixed link to security topic

carolinacmoravia committed:
P: Incorrect strucutre in metadata 

Hello, @tamram, 

Localization team hs reported source content issue that causes localized version to have warning alerts in localization process for article rendering. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

Commit:
incorporated some feedback

Commit:
made key and cert descriptions L3 headers

Commit:
added fixes

Commit:
added individual/group enrollment entries to cert section

Commit:
removed device certificate from definitions

Commit:
incorporated feedback

Commit:
signing chain -> cert chain

MicrosoftGuyJFlo committed:
Add verbose logging to contacting support section

Erikre committed:
Update active-directory-conditional-access-azure-portal.md

Adding Intune scenario.

v-geberr committed:
wip

maestersid committed:
Updating the Notification Hubs Troubleshooting doc to remove reference to the Classic Azure Portal

v-geberr committed:
interchangeable phrase list tutorial

v-geberr committed:
changed file name

v-geberr committed:
fix link

v-geberr committed:
wip

maestersid committed:
Adding missing images plus some Acrolinx updates

maestersid committed:
more grammar edits

maestersid committed:
further updates

maestersid committed:
one last stylistic update.

v-geberr committed:
edits

v-geberr committed:
fix link

v-geberr committed:
wip

v-geberr committed:
wip

v-geberr committed:
wip

v-geberr committed:
fixing wording

maestersid committed:
Merge pull request #2 from MicrosoftDocs/master

Pulling latest docs updates

v-geberr committed:
wording

maestersid committed:
Merge branch 'master' into classicportal_removal

v-geberr committed:
wording

maestersid committed:
small updates plus code blocks added

v-geberr committed:
wording

maestersid committed:
further edits plus code block corrections

maestersid committed:
correcting code blocks

v-geberr committed:
reword after Kamran's suggestions

maestersid committed:
trying to get this code block to work

v-geberr committed:
wip

maestersid committed:
final small tweak

v-geberr committed:
wip

v-geberr committed:
wip

v-geberr committed:
wip

v-geberr committed:
wip

maestersid committed:
removing references to GCM and replacing with FCM

maestersid committed:
review edits updated, removed PII in image and adjusted spacing

maestersid committed:
adjusted spacing to hopefully correct indentation issues

Commit:
sync with master and resolved conflict in articles/iot-dps/toc.yml

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-revoke-enrollment-portal
sync with master to pick up last QS changes

Commit:
added revoke device access back in to TOC

maestersid committed:
trying something different

jeffreyjirwin committed:
Update storage-blobs-introduction.md

Updating to include GPv2.

maestersid committed:
one last spacing attempt?

jeffreyjirwin committed:
Update storage-blobs-introduction.md

Commit:
task 343: Separate match conditions to reference area and link to them

Commit:
task 343: fix missing line

MikeDodaro committed:
Update image supported mkts. Add Next Steps to image and news supported mkts pages.

MikeDodaro committed:
Add Next Steps to image supported mkts

Commit:
fixed pub date

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-revoke-enrollment-portal
sync with master

mayanknayar committed:
Network interface management for ASR

Article on network interface management for ASR

mayanknayar committed:
Edit for network interface article

Readability improvements

mayanknayar committed:
ToC update for network interfaces

ToC update for network interfaces

Commit:
task 343: fix edit issues, add Always, Device match conditions to reference

maestersid committed:
trying again to correct alignment

MikeDodaro committed:
Take out sections added by error

MikeDodaro committed:
Fix link text in endpoints doc

rjagiewich committed:
Merge pull request #26737 from JimacoMS2/dps-revoke-enrollment-portal

DPS Revoke device access using the portal

maestersid committed:
more hands on using html for formatting

maestersid committed:
another formatting fix

rjagiewich committed:
Merge pull request #30566 from dksimpson/dean-343

task 343: Separate match conditions to reference area and link to them

rjagiewich committed:
Merge pull request #30388 from Erikre/patch-5

Update active-directory-conditional-access-azure-portal.md

maestersid committed:
I have no idea anymore

GitHubber17 committed:
Merge pull request #30574 from mayanknayar/master

Network interface management for ASR

rjagiewich committed:
Merge pull request #30563 from jeffreyjirwin/patch-1

Update storage-blobs-introduction.md

v-geberr committed:
review changes

maestersid committed:
found an individual whitespace

maestersid committed:
removing forced formatting

GitHubber17 committed:
Merge pull request #29298 from carolinacmoravia/patch-498

P: Incorrect strucutre in metadata

rjagiewich committed:
Merge pull request #30573 from MikeDodaro/updates-supported-mkts

Updates supported mkts

PRMerger12 committed:
Merge pull request #28471 from carolinacmoravia/patch-473

M15131: Extra ` that causing format issues

maestersid committed:
trying update with spaces instead of tabs

MicrosoftGuyJFlo committed:
Update to article per backlog item 332046

Commit:
Merge pull request #30298 from MicrosoftGuyJFlo/MFATroubleshooting

Add verbose logging to contacting support section

v-thepet committed:
remove broken art links

GitHubber17 committed:
Merge pull request #30487 from jwhiteDev/classicportal_removal

Classicportal removal

v-thepet committed:
fix NEW val warning - f1 keyword

rjagiewich committed:
Merge pull request #30579 from MicrosoftGuyJFlo/SSPR332046

Update to article per backlog item 332046

tdykstra committed:
add link to schema

Lisaco88 committed:
Merge pull request #30580 from v-thepet/linkfix

production bug: remove broken art links

rmca14 committed:
Merge pull request #30494 from v-geberr/1220-phrase-list

1220 phrase list

rmca14 committed:
Merge pull request #30581 from tdykstra/schema

add link to schema

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #30590 from MicrosoftDocs/FromPublicRepo

From public repo
